src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -29,10 +29,12 @@ public class DocumentExecuter : IDocumentExecuter
     {
         private readonly IDocumentBuilder _documentBuilder;
         private readonly IDocumentValidator _documentValidator;
+        public bool Strict { get; set; }
```

Comment:
I didn't know about the DocumentValidator, it looks like it can serve the same pupose. I'll make the change.